### PR TITLE
feat: discover tab metrics

### PR DESCRIPTION
**Pull request by sandrade-dcl**
## What does this PR change?
Add these new metrics:
- Views of places tab → `explore_places_tab_open`
- Views of events tab → `explore_events_tab_open`
- Views of favorites tab → `explore_events_tab_open`
- Clicks on Points of Interest or featured filters in places → `player_filter_places`
- Changing sort order in places → `player_sort_places`
- Clicking on featured, trending or want to go filters in events → `player_filter_events`
- Filter by event frequency, category or time → `player_filter_events`

## How to test the changes?
1. Launch the explorer.
2. Open the Explore menu.
3. Check that Places and Events sections work exactly the same way as right now in prod.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### <samp>🤖 Generated by Copilot at a7b2486</samp>

This pull request adds and improves analytics tracking for the explore V2 feature, especially for the places and events tabs and filters. It defines new filter types and events, splits existing events into more granular ones, and implements event sending for various user actions. It also refactors and fixes some scripts related to the explore V2 UI and logic. The main files affected are `PlacesAnalytics.cs`, `ExploreV2Analytics.cs`, and the component controllers and views for the places and events sub sections.

### CODE REVIEW

**Review by kwssbocian — APPROVED** [review]
✔️Discover tab works exactly the same as before. No differences in behavior observed from user side.

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/PlacesSubSectionMenu/PlacesSubSectionComponentController.cs [hunk 1]
```diff
@@ -20,6 +20,7 @@ public class PlacesSubSectionComponentController : IPlacesSubSectionComponentCon
     internal const int INITIAL_NUMBER_OF_ROWS = 4;
     private const int PAGE_SIZE = 12;
     private const string ONLY_POI_FILTER = "only_pois=true";
```
Comment (dalkia) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/PlacesSubSectionMenu/PlacesSubSectionComponentController.cs:22]: Why does this const needs the `true` in it??

Comment (dalkia) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/PlacesSubSectionMenu/PlacesSubSectionComponentController.cs:22]: Also, did you consider creating a shared enum for all this const? There is a duplication of string ids that could lead to unwanted errors

**Review by dalkia — APPROVED** [review]
Approving, with one small not critical comment

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/PlacesSubSectionMenu/PlacesSubSectionComponentController.cs [hunk 2]
```diff
@@ -134,11 +136,32 @@ private void View_OnFavoritesClicked(string placeUUID, bool isFavorite)
...
 
+    private void OpenTab()
+    {
+        exploreV2Analytics.SendPlacesTabOpen();
+        RequestAllPlaces();
+    }
+
+    private void ApplyFilters()
+    {
+        if (!string.IsNullOrEmpty(view.filter))
```
Comment (dalkia) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/PlacesSubSectionMenu/PlacesSubSectionComponentController.cs:153]: You are not sending the analytic if the filters have been cleared; if it wasnt a requirement, you can dismiss this comment.

**Pull request merged**